deploy/bundle.yaml

```diff
@@ -0,0 +1,70 @@
+---
+  apiVersion: "v1"
+  kind: "ReplicationController"
+  metadata:
+    labels:
+      name: "kubedash"
+    name: "kubedash"
+    namespace: "kube-system"
+  spec:
+    replicas: 1
+    selector:
+      name: "kubedash"
+    template:
+      metadata:
+        labels:
+          name: "kubedash"
+      spec:
+        containers:
+          - image: "gcr.io/google_containers/kubedash:v0.2.1"
+            name: "kubedash"
+            command:
+              - "/kubedash"
+              - "--headless=true"
+              - "--heapster_url=http://localhost:8082"
+            resources:
+              limits:
+                cpu: 50m
+                memory: 100Mi
+            volumeMounts:
+              - name: "ssl-certs"
+                mountPath: "/etc/ssl/certs"
+                readOnly: true
+          - image: gcr.io/google_containers/heapster:v0.18.4
```

Comment:
Because it has  never been released as a part of Kubernetes (and is probably not tested enough).